### PR TITLE
Fixes/fcm notification

### PR DESCRIPTION
**Pull request by Mubasher112**

### CODE REVIEW

File: app/services/push_notification_service.rb [hunk 1]
```diff
@@ -18,16 +18,34 @@ def send_push_notification
...
 
-    options = {notification: {title: 'Daily Notification', body: message}}
+    options = {title: 'Daily Notification', body: message}
     send(registration_ids, options)
   end
 
   private
 
+
+
```
Comment (murtazazarkhwan) [inline app/services/push_notification_service.rb]: extra lines, please remove

File: app/services/push_notification_service.rb [hunk 2]
```diff
@@ -18,16 +18,34 @@ def send_push_notification
...
   def send(registration_ids, options)
-    fcm = FCM.new(ENV['FCM_API_KEY'])
-    registration_ids.each_slice(20) do |registration_ids_slice|
-      response = fcm.send_notification(registration_ids_slice, options)
+    fcm = FCM.new(ENV['FCM_API_KEY'], Rails.root.join('config', 'credentials', 'firebase_service.json'), 'needpedia-phone-app')
```
Comment (murtazazarkhwan) [inline app/services/push_notification_service.rb:28]: can you share the 'firebase_service.json' file or provide instructions how to put this on staging and production servers? I see this is part of gitignore so I don't know how to get/create such file, what should be it's content?

**Review by murtazazarkhwan — CHANGES_REQUESTED** [review]
@Mubasher112  just some small comments

**Comment by Mubasher112** [comment]
After deployment added files on following locations of both repos


Frontend: android/app
File: google-services.json

Backend: config/credentials
File: firebase_service.json

**Pull request merged**